src/misc.ts

```diff
@@ -10,3 +10,19 @@ export function sleep(milliseconds) {
     }
   }
 }
+
+export function convertFromPixelsToRem(size: number): string {
+  return `${(size / 16).toFixed(4)}rem`;
+}
+
+export function inputValidation(object: object): Array<string> {
+  const objectValues = Object.entries(object);
+  const errorArray = [];
+  objectValues.forEach(([key, value]) => {
+    if (String(value).length < 1) {
+      errorArray.push(key);
+    }
+  });
+  return errorArray;
+}
```

Comment:
This should be in the `lib/validation` folder.

Reply:
resolved